invenio_app_ils/ui/src/invenio_app_ils/pages/backoffice/components/DeleteRecordModal/DeleteRecordModal.js

```diff
@@ -45,20 +71,14 @@ export default class DeleteRecordModal extends Component {
     }
   };
 
-  _renderActions = () => {
```

Comment:
Nothing to change, just to note that I was using the `_` just to avoid any potential method override in case in the future React new versions will have the same name (even if I highly doubt it!) :)